play-services-basement/src/main/java/org/microg/gms/common/Constants.java

```diff
@@ -28,4 +28,5 @@ public class Constants {
     public static final String MICROG_PACKAGE_SIGNATURE_SHA1 = "10321bd893f69af97f7573aafe9de1dc0901f3a1";
     @Deprecated
     public static final int MAX_REFERENCE_VERSION = GMS_VERSION_CODE;
+    public static final String GP_PACKAGE_NAME = "com.android.vending";
```

Comment:
Why not using `VENDING_PACKAGE_NAME` from [here](https://github.com/microg/GmsCore/blob/master/vending-app/src/main/java/org/microg/vending/billing/Constants.kt#L13)?

Reply:
I didn't understand your reply.
In my opinion shouldn't there be more than one costant for "com.android.vending" and also the name GP_PACKAGE_NAME isn't intuitive.

Reply:
> `VENDING_PACKAGE_NAME`

OK, got it.